### DISCUSSION TITLE
Thank you and goodbye!

### DISCUSSION THREAD

**Discussion by elalish** [google/model-viewer#4975] [General]
I'm leaving Google at the end of the year, so after 5+ years I won't be maintaining `<model-viewer>` anymore. My coworkers @diegoteran and @samaneh-kazemi will be taking over for me, and even @mrdoob keeps an eye on this project from afar. I will certainly continue to be a user of `<model-viewer>`, e.g. on my [blog](https://elalish.blogspot.com/2022/04/3d-interaction.html) and in [ManifoldCAD.org](https://manifoldcad.org/). 

Thank you all for being such a wonderful community! Certainly this wouldn't be the project it is today without all of you contributing and building things on top. Almost 100 npm projects [depend](https://www.npmjs.com/package/@google/model-viewer?activeTab=dependents) on us and [npm trends](https://npmtrends.com/@google/model-viewer-vs-aframe-vs-babylonjs) tells a pretty impressive story. We're used across hundreds of thousands of websites, adopted by the likes of NASA, VisibleBody, Shopify, Target, IKEA, and Walmart.

I like to think our discussions forum is now a pretty rich resource, not just for `<model-viewer>`, but also for glTF, AR, 3D modeling, and even some web design. I'm so glad to see you all answering each other's questions more and more - it gives me hope you can take up the slack when I'm gone. And feel free to advertise and solicit services (politely); I think answering people's questions is an excellent way to drum up business. We believe strongly in building businesses around open source code.

If you want to keep in touch, I'll continue to be active on Github, particularly my personal project: [Manifold](https://github.com/elalish/manifold). Or you can always hunt me down on [LinkedIn](https://www.linkedin.com/in/emmett-lalish/). But please, no `<model-viewer>` questions 😄.

**Comment by danilokrulj** [comment]
I'm sad to hear this, only because this discussions forum wont be the same without you :(
Thank you for everything you did here and all the times you helped people find solutions to their problems! Best of luck in life and your future projects!

**Comment by milesgreen** [comment]
This is a bit overdue, but I just wanted to take a moment to thank @elalish for the incredible work and dedication to this project. Your work has been invaluable to the community and has had a huge impact on the web and XR as a whole. We truly appreciate all the support you've given us along the way!

**Comment by b0j-an** [comment]
i could write a novel here, but in short your work changed my life, thank you

**Comment by panghuohz** [comment]
Goodbye, thank you
